.github/workflows/vipcs.yml

```diff
@@ -0,0 +1,38 @@
+name: VIPCS
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    paths:
+      - '**/*.php'
+      - 'phpcs.xml*'
```

Comment:
Added this as edits to PHPCS config files could change  VIPCS behaviour or excludes could hide files.